maltoolbox/language/languagegraph.py

```diff
@@ -497,6 +504,34 @@ class LanguageGraphAttackStep:
     def __hash__(self):
         return hash(self.full_name)
 
+    @property
+    def children(self) -> dict[
+            str,
+            list[tuple[LanguageGraphAttackStep, Optional[ExpressionsChain]]]
+        ]:
+        """
+        Get all (both own and inherited) children of a LanguageGraphAttackStep
+        """
+
+        if self.overrides:
+            # Override overrides the children
+            return self.own_children
+
+        inherited_children: dict[
+            str,
+            list[tuple[LanguageGraphAttackStep, Optional[ExpressionsChain]]]
+        ] = {}
+
+        if self.inherits:
+            inherited_children |= self.inherits.children
+        inherited_children |= self.own_children
```

Comment:
Just initialize it with self.own_children directly, it looks a bit cleaner and it makes intuitive sense.